### PR TITLE
Replace jellyfin-plugin-media-analyzer with its updated fork jellyfin-intro-skipper

### PR DESCRIPTION
**Pull request by Hadyark**
Jellyfin Media Analyzer asks on its readme to use its fork, so you might as well replace it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 - [FinTube](https://github.com/AECX/FinTube) - Easily add content from YouTube to your Jellyfin installation.
 - [InPlayerEpisodePreview](https://github.com/Namo2/InPlayerEpisodePreview) - Adds an episode list to the video player.
 - [jellyfin-ani-sync](https://github.com/vosmiic/jellyfin-ani-sync) - Automatically tracks anime watching progress on [Anilist](https://anilist.co/).
-- [jellyfin-plugin-media-analyzer](https://github.com/endrl/jellyfin-plugin-media-analyzer) - Fingerprint audio to automatically detect intro and outro segments in Jellyfin.
+- [jellyfin-intro-skipper](https://github.com/jumoog/intro-skipper) - Fingerprint audio to automatically detect intro and outro segments in Jellyfin.
```
Comment (darmiel) [inline README.md:28]: Let's go with the ‘official’ repo name :) Please also update the sorting accordingly, thanks!

```suggestion
- [intro-skipper](https://github.com/jumoog/intro-skipper) - Fingerprint audio to automatically detect intro and outro segments in Jellyfin.
```

**Review by darmiel — CHANGES_REQUESTED** [review]
Good point 👍🏻

**Comment by Hadyark** [comment]
Done

File: README.md (hunk 1 above)
Comment (Hadyark) [reply README.md]: > Let's go with the ‘official’ repo name :) Please also update the sorting accordingly, thanks!

Done

**Review by darmiel — APPROVED** [review]

**Review by TomRomeo — APPROVED** [review]

**Pull request merged**